core/domain/exp_fetchers.py

```diff
@@ -673,3 +676,47 @@ def get_exploration_version_history(
         version_history_model.metadata_last_edited_committer_id,
         version_history_model.committer_ids
     )
+
+
+def get_user_progress_in_exploration(
+    user_id: str, exp_ids: List[str]
+) -> List[exp_domain.ExplorationProgressSummaryDict]:
+    """Returns the progress of multiple users in multiple exploration.
```

Comment:
Isn't it just for a single user? I only see one user_id being passed?

Reply:
Yes, it's a single user. Updated PTAL